fe/fe-core/src/main/java/org/apache/doris/qe/GlobalVariable.java

```diff
@@ -104,7 +110,22 @@ public final class GlobalVariable {
     @VariableMgr.VarAttr(name = SHOW_FULL_DBNAME_IN_INFO_SCHEMA_DB, flag = VariableMgr.GLOBAL)
     public static boolean showFullDbNameInInfoSchemaDb = false;
 
-    // Don't allow to create instance.
+    @VariableMgr.VarAttr(name = SQL_CONVERTER_SERVICE, flag = VariableMgr.GLOBAL)
+    public static String sqlConverterService = "";
```

Comment:
Maybe `sqlConverterServiceUrl` fits the meaning better? just my personal preference

Reply:
done